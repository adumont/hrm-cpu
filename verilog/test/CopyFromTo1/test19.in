22 75
53 38
48 7d
16 5c
51 3b
22 a6
35 67
49 35
44 1a
38 24
05 72
57 83
09 dc
26 e6
25 a4
05 02
37 01
58 a0
18 2a
42 67
38 93
09 dd
30 48
13 10
17 47
04 1a
31 a4
44 67
52 41
57 b2
34 b8
14 d9
57 02
58 ec
08 7d
33 2d
58 71
02 33
31 a0
54 3b
25 c3
