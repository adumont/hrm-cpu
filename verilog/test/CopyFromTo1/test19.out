75
38
7d
5c
3b
a6
67
35
1a
24
72
83
dc
e6
a4
02
01
a0
2a
67
93
dd
48
10
47
1a
a4
67
41
b2
b8
d9
02
ec
7d
2d
71
33
a0
3b
